powershell/plugins/plugin-tweak-model.ts

```diff
@@ -97,6 +97,14 @@ async function tweakModelV2(state: State): Promise<PwshModel> {
 
   for (const group of values(model.operationGroups)) {
     for (const operation of values(group.operations)) {
+      for (const response of values(operation.responses)) {
+        // Mark returned object in response
+        let respSchema: Schema = (response as any).schema;
+        if (respSchema?.type === SchemaType.Object) {
+          respSchema.extensions = respSchema.extensions || {}
```

Comment:
extension node is for x-ms-* extensions. I think you may put it under resSchema directly.